www/userguide/Makefile.manual

```diff
@@ -0,0 +1,9 @@
+all: userguide
+
+# pack binary
```

Comment:
The comments in these files are useless, and derailing. Why is this called `Makefile.manual` instead of just simply `Makefile`?